_extras/checkout.md

```diff
@@ -102,44 +94,37 @@ Please note that:
     ~~~
     {: .source}
 
-5.  You only have to submit *one* change to move on to the next step,
-    not one per Carpentry.
+5.  We do *not* want exercises or other material that introduce new concepts:
+    our lessons are already full to bursting. If you are a Software Carpentry instructor and are
+    submitting an exercise, we will accept contributions to our [new Software Carpentry Python lesson]({{ site.swc_github}}/python-novice-gapminder/).
```

Comment:
I suggest leaving out reference to the python lesson here. You have this duplicated with the list of lessons currently accepting contributions below.